.gitignore

```diff
@@ -1,4 +1,5 @@
 /target/
+/buck-out/
```

Comment:
I would put this on the bottom and add a comment: `# To make working on ... easier`, perhaps with an issue id.

Reply:
Done. I also went ahead and made the capitalization in comments here consistent.